test/mpi/test_mpi.cc

```diff
@@ -54,16 +57,31 @@ UccTestMpi::UccTestMpi(int argc, char *argv[], ucc_thread_mode_t _tm, int is_loc
     };
     tm = _tm; //TODO check ucc provided
     /* Init ucc context for a specified UCC_TEST_TLS */
-    ucc_context_params_t ctx_params = {
-    };
+    ucc_context_params_t ctx_params          = {};
+    ucc_context_params_t onesided_ctx_params = {};
     if (!is_local) {
-        ctx_params.mask            |= UCC_CONTEXT_PARAM_FIELD_OOB;
+        ctx_params.mask         |= UCC_CONTEXT_PARAM_FIELD_OOB;
         ctx_params.oob.allgather = oob_allgather;
         ctx_params.oob.req_test  = oob_allgather_test;
         ctx_params.oob.req_free  = oob_allgather_free;
         ctx_params.oob.coll_info = (void*)(uintptr_t)MPI_COMM_WORLD;
         ctx_params.oob.n_oob_eps = size;
         ctx_params.oob.oob_ep    = rank;
+
+        onesided_ctx_params = ctx_params;
+        for (auto i = 0; i < UCC_TEST_N_MEM_SEGMENTS; i++) {
+            onesided_buffers[i] = ucc_calloc(UCC_TEST_MEM_SEGMENT_SIZE, size);
```

Comment:
ucc_calloc takes 3rd arg as "str". it works now because we don't use ucm, but may fail to compile in the future